staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/typeconverter_test.go

```diff
@@ -26,17 +26,17 @@ import (
 	"sigs.k8s.io/yaml"
 
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal"
+	"k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager"
 	"k8s.io/kube-openapi/pkg/util/proto"
 	prototesting "k8s.io/kube-openapi/pkg/util/proto/testing"
 )
 
-var fakeSchema = prototesting.Fake{
+var fakeSchema2 = prototesting.Fake{
```

Comment:
Do we need 2 fake schemas, or are they the same?

Reply:
The schemas are different, one from [api/openapi-spec/swagger.json](https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/swagger.json) and one from [testdata/swagger.json](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/testdata/swagger.json). It's possible that they can be shared but I'm not familiar enough with what they're used for.

Reply:
OK, can we rename these maybe? "testSchema" or "openapiSchema" so that we know what they are?

Reply:
ack, sounds good!